From 26645ebfb0d8a7b54acfb4ee6be15cf14e434e20 Mon Sep 17 00:00:00 2001
From: Saahil Tomar <quic_saahtoma@quicinc.com>
Date: Thu, 18 Apr 2024 11:25:11 +0530
Subject: [PATCH] net: disable Werror temporarily

This change disables Werror for the net directory temporarily.
The resolution of these net/ directory warnings requires an
extended timeframe.

Change-Id: Ib2173c7afdd70b2c36bffdd9bc17945c82f72bc5
Signed-off-by: Saahil Tomar <quic_saahtoma@quicinc.com>
---
 net/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/Makefile b/net/Makefile
index eac547aca3b4..26d8d19fed79 100644
--- a/net/Makefile
+++ b/net/Makefile
@@ -5,7 +5,7 @@
 # 2 Sep 2000, Christoph Hellwig <hch@infradead.org>
 # Rewritten to use lists instead of if-statements.
 #
-
+KBUILD_CPPFLAGS := $(filter-out -Werror, $(KBUILD_CPPFLAGS))
 obj-y				:= devres.o socket.o core/
 
 obj-$(CONFIG_COMPAT)		+= compat.o
-- 
2.34.1

